src/LinAlg/hiopVectorRaja.hpp

```diff
@@ -306,18 +309,28 @@ class hiopVectorRajaPar : public hiopVector
   virtual bool is_equal(const hiopVector& vec) const;
   
 private:
+  ExecSpace<MEMBACKEND, EXECPOLICYRAJA> exec_space_;
+  using MEMBACKENDHOST = typename MEMBACKEND::MemBackendHost;
+
+  //EXECPOLICYRAJA is used internally as a execution policy. EXECPOLICYHOST is not used internally
+  //in this class. EXECPOLICYHOST can be any host policy as long as memory allocations and
+  //and transfers within and from `exec_space_host_` work with EXECPOLICYHOST (currently all such
+  //combinations work).
```

Comment:
This comment is very helpful, and makes me wonder if we should be focusing more on HiOp documentation/user manual in order to communicate these sorts of things. I admit I am not as up to date on the sate of HiOp docs as others however.

Reply:
for sure will be included in the docs for the next release